hack/update-generated-protobuf.sh

```diff
@@ -31,19 +31,13 @@ function prereqs() {
   fi
   kube::build::ensure_docker_daemon_connectivity || return 1
 
-  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}/go-to-protobuf")
+  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}")
```

Comment:
I think this was copied from the code for releases at the time to get a
unique value.  And possibly to avoid rebuilding if that was not necessary?

On Thu, Jun 30, 2016 at 7:16 PM, Tim Hockin <notifications@github.com>
wrote:

> In hack/update-generated-protobuf.sh
> <https://github.com/kubernetes/kubernetes/pull/25525#discussion_r69239288>
> :
>
> > @@ -31,19 +31,13 @@ function prereqs() {
> >    fi
> >    kube::build::ensure_docker_daemon_connectivity || return 1
> >
> > -  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}/go-to-protobuf")
> > +  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}")
>
> thoughts on this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25525/files/70ad115beafcf9bd93386a66e1f8d5d6b9588331#r69239288>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p_Sn21LQt7HfHYt6I7iwK8413CX2ks5qRHiIgaJpZM4Icxfh>
> .
>